packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.5+3
+
+* CameraPlatform.createCameraWithSettings to allow recorded video fps and bitrate control.
```

Comment:
`CameraPlatform.createCameraWithSettings` isn't included in the app facing package. This should reference `CameraController.withSettings`. Something like: 

```suggestion
* Adds support to control video fps and bitrate. See `CameraController.withSettings`.
```

Reply:
Committed suggestion.